### ISSUE TITLE
vulns reported on build

### ISSUE THREAD

**Issue by significance** [ethersphere/bee-dashboard#302]
```
107 vulnerabilities (94 moderate, 11 high, 2 critical)

To address issues that do not require attention, run:
  npm audit fix

To address all issues (including breaking changes), run:
  npm audit fix --force

Run `npm audit` for details.
npm notice
npm notice New major version of npm available! 7.7.6 -> 8.5.0
npm notice Changelog: https://github.com/npm/cli/releases/tag/v8.5.0
npm notice Run npm install -g npm@8.5.0 to update!
npm notice
ip-192-168-0-57 :: Code/swarm/bee-dashboard » npmv
ip-192-168-0-57 :: Code/swarm/bee-dashboard 130 » git remote -v
origin	git@github.com:ethersphere/bee-dashboard.git (fetch)
origin	git@github.com:ethersphere/bee-dashboard.git (push)
ip-192-168-0-57 :: Code/swarm/bee-dashboard » npm -v
7.7.6
ip-192-168-0-57 :: Code/swarm/bee-dashboard » node -v
v15.14.0
```

**Comment by vojtechsimetka** [comment]
blocked by #311 (need to merge it first to avoid conflicts)

**Comment by vojtechsimetka** [comment]
So sadly this is actually not really trivial and the true solution is to replace the nodejs dependencies that do not have browser implementation with a browser one. This way we don't have to use polyfills and switch to `react-app-rewired`. See #313

**Comment by BalintSP** [comment]
Thank you for the report.

We have investigated and fixed this issue. The resolution is included in bee-dashboard [v0.36.1](https://github.com/ethersphere/bee-dashboard/releases/tag/v0.36.1) and Swarm-desktop [v0.55.1](https://github.com/ethersphere/swarm-desktop/releases/tag/v0.55.1).
Please update to these latest versions (or newer) to get the fix.

If the issue still occurs on your side, please reopen it or create a new issue with the following details:
- exact steps to reproduce
- actual result
- expected result
- browser 
- screenshots or screen recording, log if possible

Thank you for helping us improve the product!
